index.html

```diff
@@ -21,145 +22,161 @@
-					</ul>
-				</section>
-				<footer class="footerCards">
-					<button type="button">See Project</button>
-				</footer>
+			<div class="cardsGrid">
+				<div class="cards">
+					<section class="contentCards">
+						<h1>Profesional Art Printing Data</h1>
+						<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+							industry's standard</p>
+						<ul>
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</section>
+					<footer class="footerCards">
+						<button type="button">See Project</button>
+					</footer>
+				</div>
+				<div class="cards">
+					<section class="contentCards">
+						<h1>Profesional Art Printing Data</h1>
+						<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+							industry's standard</p>
+						<ul>
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</section>
+					<footer class="footerCards">
+						<button type="button">See Project</button>
+					</footer>
+				</div>
+				<div class="cards">
+					<section class="contentCards">
+						<h1>Profesional Art Printing Data</h1>
+						<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+							industry's standard</p>
+						<ul>
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</section>
+					<footer class="footerCards">
+						<button type="button">See Project</button>
+					</footer>
+				</div>
+				<div class="cards">
+					<section class="contentCards">
+						<h1>Profesional Art Printing Data</h1>
+						<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+							industry's standard</p>
+						<ul>
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</section>
+					<footer class="footerCards">
+						<button type="button">See Project</button>
+					</footer>
+				</div>
+				<div class="cards">
+					<section class="contentCards">
+						<h1>Profesional Art Printing Data</h1>
+						<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+							industry's standard</p>
+						<ul>
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</section>
+					<footer class="footerCards">
+						<button type="button">See Project</button>
+					</footer>
+				</div>
+				<div class="cards">
+					<section class="contentCards">
+						<h1>Profesional Art Printing Data</h1>
+						<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+							industry's standard</p>
+						<ul>
+							<li>Html</li>
+							<li>Css</li>
+							<li>Javascript</li>
+						</ul>
+					</section>
+					<footer class="footerCards">
+						<button type="button">See Project</button>
+					</footer>
```

Comment:
Hello Diego, I would like to suggest you checking if we can use this footer inside de cards... as I thought it is for the footer of the page.